eng/vmr-sync.ps1

```diff
@@ -0,0 +1,212 @@
+<#
```

Comment:
Question - do we have a PR leg in installer that will use this script? Just to have the coverage?

Reply:
yes - this PR in fact :)

Reply:
I only see the Linux one being executed. Is this about some additional pipeline you run via `/azp run`?